File: 351003/Kisel/NoSQL/RV_Kisel_lab3_Task330/Models/Entities/Creator.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace RV_Kisel_lab2_Task320.Models.Entities;

[Table("tbl_creator")]
public class Creator {
    [Key]
    [Column("id")] // <--- ДОБАВИТЬ ЭТО!
    public int Id { get; set; }

    [Column("login")] // <--- И для остальных колонок тоже!
    public string Login { get; set; } = string.Empty;

    [Column("password")]
    public string Password { get; set; } = string.Empty;

    [Column("firstname")]
    public string Firstname { get; set; } = string.Empty;

    [Column("lastname")]
    public string Lastname { get; set; } = string.Empty;

    public ICollection<News> News { get; set; } = new List<News>();
}

File: 351001/Sosnovskiy/module-publisher/src/main/java/by/egorsosnovski/distcomp/entities/Creator.java
package by.egorsosnovski.distcomp.entities;

import jakarta.persistence.*;
import jakarta.validation.constraints.Size;
import lombok.*;

@Entity
@Getter
@Setter
@ToString
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "tbl_creator")
public class Creator {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    long id;

    @Column(unique = true)
    @Size(min = 2, max = 64)
    String login;

    @Size(min = 8, max = 128)
    String password;

    @Size(min = 2, max = 64)
    String firstname;

    @Size(min = 2, max = 64)
    String lastname;
}
